test/jasmine/tests/geo_test.js

```diff
@@ -1390,6 +1555,8 @@ describe('Test geo interactions', function() {
             .catch(failTest)
             .then(done);
         });
+
+        // TODO add test for scope:'none'
```

Comment:
Would you be interested to add these tests in this PR?

Reply:
Yes, when I'll get to this TODO item

> - [ ] Add some "scope-none" option for @nicolaskruchten, which would hide all the base layers using a single setting. Thinking about this in more details today, I think it would be better to add a geo.visible attribute similar to axis.visible instead.